src/ducks/index.js

```diff
@@ -1,10 +1,10 @@
 import { combineReducers } from 'redux'
 
-import { myAppsReducers } from '../ducks/myApps'
+import { appsReducers } from '../ducks/myApps'
 import alerterReducer from 'cozy-ui/react/Alerter'
 
 const storeApp = combineReducers({
-  myApps: myAppsReducers,
+  apps: appsReducers,
   alerts: alerterReducer
```

Comment:
What about:
```js
import { combineReducers } from 'redux'

import apps from './apps'
import currentAppVersion from './apps/currentAppVersion'
import alerts from 'cozy-ui/react/Alerter'

const storeApp = combineReducers({
  apps
  currentAppVersion,
  alerts
})

export default storeApp
```
Ducks are supposed to export a default function named `reducer` https://github.com/erikras/ducks-modular-redux